enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/constants.ts

```diff
@@ -12,7 +12,11 @@ import type {
 /** The maximum number of recent items to show in the resource selection step. */
 export const EMBED_RESOURCE_LIST_MAX_RECENTS = 6;
 
-export const getEmbedExperiences = () =>
+export const getEmbedExperiences = ({
```

Comment:
do you plan to call this function outside React? Otherwise, you could make this a hook and don't need to pass anything. Anyways, that's just another suggestion, but I think the current solution also works.